src/run.js

```diff
@@ -58,6 +58,7 @@ const getSha = (eventName, payload) => {
 const run = async (context) => {
   const eventName = context != null ? context.eventName : github.context.eventName
   const payload = context != null ? context.payload : github.context.payload
+  const login = payload.sender ? payload.sender.login : ''
```

Comment:
I haven't look up the documentation myself, but could `sender`  be `undefined` or `null`?

Reply:
Tks Nitish, yes the sender property is included in every webhook payload.